public/app/plugins/datasource/prometheus/querybuilder/components/LabelFilters.tsx

```diff
@@ -0,0 +1,81 @@
+import { isEqual } from 'lodash';
```

Comment:
These are identical except for a single property: https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/querybuilder/shared/LabelFilters.tsx. 
😢